index.bs

```diff
@@ -1644,6 +1644,18 @@ than U+000A LINE FEED (LF) characters, U+000D CARRIAGE RETURN (CR) characters, U
 characters (&amp;), and U+003E GREATER-THAN SIGN characters (>).</p>
 
 
+<h4 id=chapter-title-text>WebVTT chapter title text</h4>
+
+<p><dfn>WebVTT chapter title text</dfn> is <a>WebVTT cue text</a> that makes use only of zero or
+more of the following components, each optionally separated from the next by a <a>WebVTT line
```

Comment:
If the "only" applies to the following components, maybe rephrase as "makes use of zero or more of the following components, and no others, each ..."

Reply:
@nigelmegitt what is the alternative way of interpreting that? What's the source of confusion?

Reply:
@silviapfeiffer it looks as though the "only" applies to the number of components, not the list from which that number could be selected. Saying "only of zero or more" looks odd, because it would be more natural to say "of zero or more", or "of any number". "Only" seems a bit of a sticky word, in that it feels like it is a qualifier that applies to the adjacent thing, rather than something later in the sentence. So if you say "of zero of more of the following components only, ..." then the "only" clearly applies to "the following components".

Reply:
I see. Maybe we could just remove the word "only"?

Reply:
Yes, that would work.